game/functions/systems/skills/client/network/fn_skills_requestSkillRespec.sqf

```
@@ -20,6 +20,6 @@
 
 params [["_displayParent", nil]];
 
-["Waiting for server...", "Please wait", false, false, _displayParent] spawn BIS_fnc_guiMessage;
+isNil {["Waiting for server...", "Please wait", false, false, _displayParent] spawn BIS_fnc_guiMessage};
```

Why does this one need to be unscheduled? Don't mind the change, just curious.

---

When `fn_skills_requestSkillRespec` is called from scheduled this line will warn/error about `_displayParent` being `nil` so this basically silences the error dialog.